peer.go

```diff
@@ -223,6 +223,9 @@ func newPeer(conn net.Conn, connReq *connmgr.ConnReq, server *server,
 	return p, nil
 }
 
+// A compile-time check to ensure that peer satisfies the FundingPeer interface.
+var _ FundingPeer = (*peer)(nil)
```

Comment:
Hm, this has the unfortunate effect of keeping the dependency cycle `peer<->fundingmanager`. Why not make `AddNewChannel` part of the regular `Peer` interface?

Reply:
Fixed.